### ISSUE TITLE
K.categorical_crossentropy increase with different version of keras and put the performance down.

### ISSUE THREAD

**Issue by Lan1991Xu** [keras-team/keras#8125]
Hi, I try to figure out why the keras version from 1.1.1 to 2.0 will decrease the performance of this code https://github.com/titu1994/Snapshot-Ensembles. 
My first environment is keras 1.1.1. tensorflow 0.12
Seconde is keras 2.0 tensorflow 1.0
The performance is drop 2 %.
The more confused me is when I design the custom_crossentropy, just use the K.categorical_crossentrop. It is around 15, basicly it is around 5. Can anyone give me the explain for this situation?

**Comment by Lan1991Xu** [comment]
More important. I find the loss increased is not by tensorflow version change. It is by keras change, when I use version of 1.X, the loss is three time than version of 2.0. Both of them are customer loss.

**Comment by ymodak** [comment]
Closing this issue due to staleness. Please use the latest version of TensorFlow and Keras and test again. Feel free to reopen this issue if the issue still persists.
